bliss/conf/base_config.yaml

```diff
@@ -165,13 +165,16 @@ predict:
         bands: ${encoder.bands}
         predict_device: ${predict.device}
         predict_crop: ${predict.crop}
-    photo_catalog:
-        _target_: bliss.surveys.sdss.PhotoFullCatalog
-        sdss_dir: ${paths.sdss}
-        run: ${predict.dataset.run}
-        camcol: ${predict.dataset.camcol}
-        fields: ${predict.dataset.fields}
-        bands: ${predict.dataset.bands}
+    # dataset:
+    #     _target_: bliss.surveys.decals.DarkEnergyCameraLegacySurvey
+    #     decals_dir: ${paths.decals}
+    #     ra: 336.635 # in degrees
+    #     dec: -0.96 # in degrees
+    #     width: 2400
+    #     height: 1489
+    #     bands: ["g", "r", "z"]
+    #     predict_device: ${predict.device}
+    #     predict_crop: ${predict.crop}
     trainer: ${training.trainer}
     encoder: ${encoder}
```

Comment:
We'd also likely need a simulator to generate data that resembles decals data, i.e. which samples backgrounds and PSFs from DECaLs data. (The simulator above has an `sdss_fields` entry -- but ideally the simulator wouldn't be specific to any one survey.)

Reply:
DECaLS data generation - punting to after #853.

Reply:
tracked by #871